### ISSUE TITLE
[Feature Request] Start minimized to tray

### ISSUE THREAD

**Issue by erikw** [aluxian/Messenger-for-Desktop#1328]
* Operating system: Arch linux
* Messenger for Desktop version:`2.0.9+42+g4e4d6f1-2`

Hello,

I launch messengerfordesktop manually in my `~/.xinitrc` file, as I load my X system with `startx(1)`. I would love it there was a option to make this program start minimized to the tray, as I don't want to see it when I log in to my computer, only when I want to directly use it.

I see in the App-menu that there is a grayed out option "Start hidden on Startup", which I guess works with "Launch on OS startup" option, which I'm not using (I'm not using a giant desktop environment like Gnome or KDE that has support for these startup applications, I manage this myself with my xinitrc script).

TL;DR
It would be awesome to have this program to start minimized, unrelated to if you use "Launch on OS startup" or not.

=)

**Comment by beppe9000** [comment]
as a workaround you can use something like autohotkey/autoit to script the minimization at startup

**Comment by erikw** [comment]
@beppe9000 Thanks! Unfortunately I think those tools are for windows only, not Linux :)

**Comment by beppe9000** [comment]
@erikw have a look [here](https://github.com/Paris/IronAHK) or [here](http://www.semicomplete.com/projects/xdotool/xdotool.xhtml)

**Comment by erikw** [comment]
Ah I found a soltuion

1. Configure by going to the App menu, and check both "Launch on OS startup" and "Start hidden on Startup"
2. Launch the app like `messengerfordesktop --os-startup`

This is a solution that works!


Just for the record, I tried xdotool,

```bash
# Does not work
$ xdotool windowminimize $(xdotool search --name 'Messenger for Desktop')
# This closes the window, but Messengerfordesktop stops responding and can't be opended from tray again.
$ xdotool windowclose $(xdotool search --name 'Messenger for Desktop')
```

**Comment by beppe9000** [comment]
good. i do not use that parameter on windows currently because the app "forgets" its window state and needs to be maximized with external tools and then closed to be put in tray.
